File: DotNetUEApp_C#/Models/ActiviteApprentissage.cs
using System.ComponentModel.DataAnnotations;

namespace DotNetUEApp_C_.Models;


public class ActiviteApprentissage
{
    public int Id { get; set; }
    public string Name { get; set; }


    public Evaluation? EvaluationAA { get; set; }

   // public List<Enseignant>? ListEnseignant { get; set; }


}

File: models/activiteApprentissage.ts
// activite-apprentissage.ts (dans le dossier models)

import { Enseignant } from "./enseignant";
import { Evaluation } from "./evaluation";

export class ActiviteApprentissage {
    id!: number;
    name!: string;
    idUniteEtude!: number | null;
    evaluationAA!: Evaluation | null; // Si vous définissez le modèle pour Evaluation
  }
  

  